updatecli/updatecli.d/updatecoredns.yaml

```diff
@@ -0,0 +1,74 @@
+---
+name: "Update coredns version" 
+
+sources:
+ coredns:
+   name: Get coredns version
+   kind: githubrelease
+   spec:
+     owner: coredns
+     repository: coredns
+     token: '{{ requiredEnv .github.token }}'
+     typefilter:
+       release: true
+       draft: false
+       prerelease: false
+     versionfilter:
+       kind: latest
+
+targets:
+  dockerfile:
+    name: "Bump to latest coredns version in Dockerfile"
+    kind: dockerfile
+    scmid: default
+    sourceid: coredns
+    spec:
+      file: "Dockerfile"
+      instruction:
+        keyword: "ARG"
+        matcher: "TAG"
```

Comment:
Need to check if this line will also not be bumped https://github.com/rancher/image-build-coredns/blob/master/Dockerfile#L18, together with the intended one https://github.com/rancher/image-build-coredns/blob/master/Dockerfile#L2.